docs/src/tutorial.md

```diff
@@ -103,10 +103,20 @@ That way, the accuracy will not fluctuate with the minibatch.
 
 ## Default behavior
 By default, the training minibatch that evaluates the neural network doesn't change between evaluations.
-To change the training minibatch, use:
+To change the training minibatch, use one of the following methods:
+* To select randomly a mini-batch 
+```@example KnetNLPModel
+rand_minibatch_train!!(DenseNetNLPModel)
```

Comment:
```suggestion
rand_minibatch_train!(DenseNetNLPModel)
```
I think `!!` makes the documentation fails inevitably.

Reply:
I let you commit the suggestion, because I don't want to mess your local branch if you made changes since your last git push.

Reply:
oh, nice catch